04.ls/ls.rb

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env ruby
+
+@dir = Dir.glob '*'
```

Comment:
このインスタンス変数は、このプログラムのどこからでも参照ができる実質「グローバル変数」となっていますね。 `@dir` に何が入っているか常に意識しながらコードを読み書きする必要があり、コードが読みにくくなってしまうので、こういった使い方は避けていただきたいです。

Reply:
グローバル変数はやめて変数にいたしました。
```suggestion
#!/usr/bin/env ruby

dir_origin = Dir.glob '*'
```